features/supplier/opportunities.feature

```diff
@@ -102,18 +102,43 @@ Scenario Outline: User can filter by both status, lot and location together
+  When I click 'All categories'
+  Then I see '<phrase>' in the search summary text
+  And I see that the stated number of results is no fewer than that result_count
+  And I note the result_count
+  And I see all the opportunities on the page are in the '<location>' location
+  When I uncheck '<location>' checkbox
+  And I wait for the page to reload
+  Then I see '<phrase>' in the search summary text
+  And I see that the stated number of results is no fewer than that result_count
 
   Examples:
-    | lot                        | status       | location                       |
-    | Digital specialists        | Open         | Scotland                       |
-    | Digital outcomes           | Open         | International (outside the UK) |
-    | User research participants | Open         | Off-site                       |
-    | Digital specialists        | Closed       | South West England             |
-    | Digital outcomes           | Closed       | International (outside the UK) |
-    | User research participants | Closed       | Off-site                       |
-    | Digital specialists        | Open         | London                         |
-    | Digital outcomes           | Closed       | International (outside the UK) |
-    | User research participants | Open         | Off-site                       |
-    | Digital specialists        | Closed       | Wales                          |
-    | Digital outcomes           | Open         | Yorkshire and the Humber       |
-    | User research participants | Closed       | Northern Ireland               |
+    | lot                        | status       | location                       | phrase              |
+    | Digital specialists        | Open         | Scotland                       | governments         |
+    | Digital outcomes           | Open         | International (outside the UK) | digital             |
+    | User research participants | Open         | Off-site                       | services            |
+    | Digital specialists        | Closed       | South West England             | enterprise software |
+    | Digital outcomes           | Closed       | International (outside the UK) | delivery            |
+    | User research participants | Closed       | Off-site                       | "agile methodology" |
+    | Digital specialists        | Open         | London                         | improve             |
+    | Digital outcomes           | Closed       | International (outside the UK) | performance         |
+    | User research participants | Open         | Off-site                       | analyze             |
+    | Digital specialists        | Closed       | Wales                          | management          |
+    | Digital outcomes           | Open         | Yorkshire and the Humber       | national            |
+    | User research participants | Closed       | Northern Ireland               | metempsychosis      |
```

Comment:
That we have to list all of these out in our tests feels like technical debt of some sort. If you agree then please raise a ticket, I'm not saying we should tackle that here :-) We avoided it for G-Cloud, anyway.

Reply:
Well, we're just picking out a smattering of combinations to test. It would be infeasible to test all combinations and I think it would be pretty tricky to do that in cucumber. Got to remember that cucumber is pretty limited - my `MAYBE_VAR` stuff alone was a hack pushing at its limits.